scripts/irods/test/resource_suite.py

```diff
@@ -1167,3 +1167,96 @@ def test_itrim_no_op(self):
 
         # try to trim down to repl_count
         self.admin.assert_icommand("itrim -N {repl_count} {filename}".format(**locals()), 'STDOUT_SINGLELINE', "Total size trimmed = 0.000 MB. Number of files trimmed = 0.")
+
+
+    ###################
+    # detached mode
+    ###################
+    def test_detached_mode(self):
+
+        try:
+            file1 = "f1"
+            file2 = "f2"
+            resource_host = "irods.org"
+            resource_name = 'detached_resource'
+
+            resource_context = 'HOST_MODE=detached'
+
+            lib.create_ufs_resource(resource_name, self.admin, resource_host)
+            self.admin.assert_icommand("iadmin modresc %s context %s" %
+                                   (resource_name, resource_context), 'EMPTY')
```

Comment:
I think it's safe to not specify `'EMPTY'` when you don't expect any output.

Reply:
That's true - 'EMPTY' is the default... but being explicit is okay too?

We haven't written our preference for either anywhere I don't think...

Reply:
done